src/server/service/kv.rs

```diff
@@ -1698,21 +1697,24 @@ fn future_flashback_to_version<
 ) -> impl Future<Output = ServerResult<FlashbackToVersionResponse>> {
     let storage_clone = storage.clone();
     let raft_router_clone = raft_router.clone();
+    // TODO: Make this func a two-phase request
     async move {
         // Send a `SignificantMsg::PrepareFlashback` to prepare the raftstore for the
         // later flashback. This will first block all scheduling, read and write
         // operations and then wait for the latest Raft log to be applied before
         // we start the flashback command.
         let region_id = req.get_context().get_region_id();
-        let (result_tx, result_rx) = oneshot::channel();
+        let (result_tx, result_rx) = crossbeam::channel::bounded(1);
         raft_router_clone
-            .significant_send(region_id, SignificantMsg::PrepareFlashback(result_tx))?;
-        if !result_rx.await? {
+            .significant_send(region_id, SignificantMsg::PrepareFlashback(result_tx))
```

Comment:
As it's a admin command now, no need to trigger it by significant message. We can just followe the normal way to propose admin command. Check `fn send_admin_request` for reference.